.pre-commit-config.yaml

```diff
@@ -37,8 +37,8 @@ repos:
           ]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.770
+    rev: v0.812
     hooks:
       - id: mypy
         language_version: python3.8
-        args: ["--no-strict-optional", "--ignore-missing-imports"]
+        args: ["--no-strict-optional", "--ignore-missing-imports", "--explicit-package-bases", "--namespace-packages"]
```

Comment:
ref: https://github.com/pre-commit/mirrors-mypy/issues/40#issuecomment-785000979

Reply:
Yes this is important